### ISSUE TITLE
ExLlamav3_HF loader is spitting random tokens after the recent updates

### ISSUE THREAD

**Issue by DeathGodlike** [oobabooga/text-generation-webui#7266]
### Describe the bug

Hello.

The exllamav3 hf loader stopped working properly somewhere around after october 8 updates.

How it looks like: a model starts spitting out random combinations of tokens all over the place.

### Output example

```shell
" countin"://"189997"=""

"154"
The "H"
Your

"99"

the victim13:

" - ,:"t

“` Back to his
“
```

Though, i don't know if anyone else faced this issue.

But it seems that switching to exllamav3 loader fixes the issue.

### Is there an existing issue for this?

- [x] I have searched the existing issues

### System Info

```shell
OS: Windows 10

GPU: Nvidia RTX 4070 TI Super (16 GB)
```

**Comment by oobabooga** [comment]
This should be fixed now, the commit for making exllamav3_hf work with the logits API endpoint broke its text generation so I reversed it.

**Comment by DeathGodlike** [comment]
Thank you!
